ui/src/app/components/runtime-configuration-panel/customize-panel.tsx

```diff
@@ -217,7 +217,18 @@ export const CustomizePanel = ({
                   // When the compute type changes, we need to normalize the config and potentially restore defaults.
                   setAnalysisConfig(
                     withAnalysisConfigDefaults(
-                      { ...analysisConfig, computeType },
+                      {
+                        ...analysisConfig,
+                        computeType,
+                        diskConfig: {
+                          ...analysisConfig.diskConfig,
+                          // when switching from DataProc to Standard, set to the size of any existing disk
```

Comment:
previously, it kept the DataProc disk size.  If this was below the PD size, it would prompt users to delete the PD, with no indication that it could be saved by increasing the disk size.  However, even this assumes that the system was working as intended.  The other bugs resolved in this PR prevented this method of recovering the PD as well.